telluric/georaster.py

```diff
@@ -66,30 +66,6 @@
     'jpeg': 'JPEG',
 }
 
-# source: http://wiki.openstreetmap.org/wiki/Zoom_levels
-mercator_zoom_to_resolution = {
```

Comment:
this is a breaking change

Reply:
I moved it to Constants, and gave it upper case name, 

Who is using this?

Reply:
API, and maybe others, so remember to change it on the API

Reply:
ok